src/tiled/command.h

```diff
@@ -56,6 +62,10 @@ struct Command
      */
     QString finalCommand() const;
 
+    QString finalWorkingDirectory() const;
+
+    QString replaceVariables(const QString &string, bool quotePaths = true) const;
```

Comment:
Judging by the implementation, it seems `quotePaths` should be called `quoteValues`.